partner_industry_secondary/models/res_partner_industry.py

```diff
@@ -23,16 +23,13 @@ class ResPartnerIndustry(models.Model):
     )
     parent_path = fields.Char(index=True, unaccent=False)
 
-    def name_get(self):
-        def get_names(cat):
-            """Return the list [cat.name, cat.parent_id.name, ...]"""
-            res = []
-            while cat:
-                res.insert(0, cat.name)
-                cat = cat.parent_id
-            return res
-
-        return [(cat.id, " / ".join(get_names(cat))) for cat in self]
+    @api.depends("name", "parent_id")
+    def _compute_display_name(self):
+        for rec in self:
+            if rec.parent_id:
+                rec.display_name = rec.parent_id.display_name + " / " + rec.name
```

Comment:
change it to .format .
and in line  52/49 
`default["name"] = self.name + " 2"`